interface/skimUtils.h

```diff
@@ -644,4 +644,59 @@ pair <TVector2, TVector2> getShiftedMET_mes (TVector2 MET, bigTree & theBigTree,
   return make_pair(shiftedMET_muup, shiftedMET_mudown);
 }
 
+pair <TVector2, TVector2> getShiftedMET_electrons(TVector2 met, bigTree &theBigTree, int pairType, bool isScale)
+{
+  // do nothing in the tautau channel (electrons faking taus)
+  // it is already taken into account in the getShiftedMET_tes_ees() function
+  if(pairType == 2) {
+	return make_pair(met, met);
+  }
```

Comment:
I don't think this is actually needed here. The electron faking taus corrections we only apply to reconstructed taus, the electron energy scale and resolution corrections are applied to reconstructed electrons, so there is no overlap anyway, right?

Reply:
From a chat with @kramerto, we decide that the correct thing to do is to use the reconstructed particle type: ```oph.isElectron(theBigTree.particleType->at(idx);)```.